src/style.css

```diff
@@ -2,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src:
+    url(../src/fonts/Avenir-Book.ttf) format('truetype'),
+    url(../src/fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+.body {
+  margin: 0;
+  box-sizing: border-box;
+  padding-top: 20px;
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.input {
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+  position: relative;
+  transition: box-shadow 0.3s;
+}
+
+.input__line::before {
```

Comment:
The use of the `::before` pseudo-element here is good for adding icons, but ensure that the `content` property is set correctly to work across all browsers. The `transform-box` property on line 30 is not standard and may not have the desired effect; consider using `background-position` to position the background image instead.